src/main/resources/data/spectrum/recipes/mod_integration/affinity/TODO.txt

```diff
@@ -0,0 +1 @@
+TODO:Actually play with the mod to get a good understanding of what compat to add
```

Comment:
The TODO files should probably not be part of the PR :P

Reply:
Fair, I suppose :P